frontend/ui/widget/textviewer.lua

```diff
@@ -103,38 +100,13 @@ function TextViewer:init()
         }
     end
 
-    local closeb = CloseButton:new{ window = self, padding_top = Size.padding.tiny, }
-    local title_text = TextBoxWidget:new{
-        text = self.title,
-        face = self.title_face,
-        bold = true,
-        width = self.width - 2*self.title_padding - 2*self.title_margin - closeb:getSize().w,
-    }
-    local titlew = FrameContainer:new{
-        padding = self.title_padding,
-        -- TextBoxWidget has less text top & bottom padding than TextWidget
-        -- (for a reasonable line height with multi lines), but we
-        -- can get the same as TextWidget by simply adding Size.padding.small
-        padding_top = self.title_padding + Size.padding.small,
-        padding_bottom = self.title_padding + Size.padding.small,
-        margin = self.title_margin,
-        bordersize = 0,
-        title_text
-    }
-    titlew = OverlapGroup:new{
-        dimen = {
-            w = self.width,
-            h = titlew:getSize().h
-        },
-        titlew,
-        closeb,
-    }
-
-    local separator = LineWidget:new{
-        dimen = Geom:new{
-            w = self.width,
-            h = Size.line.thick,
-        }
+    local titlebar = TitleBar:new{
+        width = self.width,
+        align = "left",
+        with_bottom_line = true,
+        title = self.title,
+        title_face = self.title_face,
```

Comment:
No need for this one ? If we want something consistent, let it use the default font in TitleBar?
Or keep this here (so callers could provide a smaller font size), but set `title_face = nil -- use default from TitleBar` on line 51 ?